src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -105,6 +105,34 @@ int PaPulseAudio_updateTimeInfo( pa_stream * s,
     return 0;
 }
 
+/* Release pa_operation always same way */
+int PaPulseAudio_ReleaseOperation(PaPulseAudio_HostApiRepresentation *hostapi,
+                                  pa_operation **operation)
+{
+    unsigned int wait = 0;
+    pa_operation *localOperation = (*operation);
+
+    while( pa_operation_get_state( localOperation ) == PA_OPERATION_RUNNING )
+    {
+        pa_threaded_mainloop_wait( hostapi->mainloop );
+
+        wait ++;
+        usleep( 1000 );
+
+        if( wait > 2000 )
```

Comment:
this condition should be checked at the top of the block, immediately after the while condition indicates still running

Reply:
Moved to correct place

Reply:
I don't see your change. Did you forget to push it?

Reply:
My bad! Not pushed stuff as you mentioned. Now done it and rebased.